### PR TITLE
Rename LICENSE to LICENSE.txt

### PR DESCRIPTION
**Pull request by jakirkham**
This is mainly to match [`MANIFEST.in`]( https://github.com/0todd0000/spm1d/blob/1aa35707b47eb19009df72b2ebb3874efd35c73f/MANIFEST.in#L1 ). Though some OSes (e.g. Windows) really like the extension too.

### CODE REVIEW

**Comment by jakirkham** [comment]
cc @melund

File: LICENSE.txt [hunk 1]
```diff
@@ -671,4 +671,4 @@ into proprietary programs.  If your program is a subroutine library, you
 may consider it more useful to permit linking proprietary applications with
 the library.  If this is what you want to do, use the GNU Lesser General
 Public License instead of this License.  But first, please read
-<http://www.gnu.org/philosophy/why-not-lgpl.html>.
\ No newline at end of file
+<http://www.gnu.org/philosophy/why-not-lgpl.html>.
```
Comment (0todd0000) [inline LICENSE.txt]: Thanks for spotting that!  I'll merge this request momentarily.

Comment (jakirkham) [inline LICENSE.txt]: Not sure I can take all the credit as it appears this was GitHub being clever. 😄

**Comment by jakirkham** [comment]
Thanks. 😄

**Pull request merged**